CONTRIBUTING.md

```diff
@@ -1016,15 +1016,6 @@ The `metric` (or equivalent) and `timeseries_limit_metric` fields are all compos
 
 The filter-box configuration references column names (via the `column` key) and optionally metric names (via the `metric` key) if sorting is defined.
 
-### Options
-
-| Field                  | Type      | Notes                                |
-| ---------------------- | --------- | ------------------------------------ |
-| `compare_lag`          | _number_  | The **Comparison Period Lag** widget |
-| `compare_suffix`       | _string_  | The **Comparison suffix** widget     |
-| `show_trend_line`      | _boolean_ | The **Show Trend Line** widget       |
-| `start_y_axis_at_zero` | _boolean_ | The **Start y-axis at 0** widget     |
-
```

Comment:
@kristw I'll just keep removing these things as the controls are moved out of `controls.js`, but let me know if you have thoughts on when/where NOT to do that, so I don't kill any useful documentation.

Reply:
I am not 100% sure about this part but seems reasonable I guess. Or you can copy the note as comment in the js file.